packages/queued-request-controller/CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3]
+### Changed
+- Bump dependency on @metamask/json-rpc-engine to ^7.2.0 ([#1895](https://github.com/MetaMask/core/pull/1895))
+- Bump @metamask/utils from 8.1.0 to 8.2.0 ([#1957](https://github.com/MetaMask/core/pull/1957))
+
+### Fixed
+- Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))
```

Comment:
What about nicknames got fixed? Is it that they're now available, and if so, available how? Also, it's not just nickname but theoretically any other network configuration information yes?